cucumber/policy/features/support/client.rb

```diff
@@ -0,0 +1,147 @@
+
+module ClientHelpers
```

Comment:
This comment applies to all the clients:

I don't think we want them under a "helpers" module, since one of the nice benefits of this approach is that this code is no longer couples to our tests, or to the "rspec" way of doing things.  I'm not sure where the best place is for them -- we can try to figure that together -- but I do think the `XXXHelpers` modules are superfluous.  

Also, to avoid name stutter, and to strengthen the conceptual connection between the clients, I'd consider a structure like:
```rb
# top level
class Client
  class Policy
  end
  
  class Resource
  end
  # etc
end
```
So then the client code we read like `Client::Policy.new(...)`, etc...